tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/debug-output-tests/main.bal

```diff
@@ -16,5 +16,5 @@
 
 // The `main` function, which acts as the entry point.
 public function main() {
-    int x = 1/0;
+    int _ = 1/0;
```

Comment:
@NipunaRanasinghe, hope this change is OK.

Reply:
@MaryamZi test results of this test is independent from the variable names so this should be fine :)